digits/extensions/view/interface.py

```diff
@@ -49,6 +49,32 @@ def get_header_template(self):
         """
         return None, None
 
+    def get_ng_templates(self):
```

Comment:
do you think we could stick to the existing `get_header_template()` (possibly we can add a `get_footer_template()` too)? It feels awkward to name APIs after the Javascript package we are using. If you move to `C3` or any other Javascript you don't want to rename the APIs right?